MusicNotationKit/MusicNotationKit/NoteDuration.swift

```diff
@@ -0,0 +1,121 @@
+//
+//  NoteDuration.swift
+//  MusicNotationKit
+//
+//  Created by Kyle Sherman on 8/20/16.
+//  Copyright © 2016 Kyle Sherman. All rights reserved.
+//
+
+public struct NoteDuration {
```

Comment:
Anything that uses TPQN will only be used in `MeasureDurationValidator` And should not leave that context. Therefore, I am exposing the information needed to do the calculation here, but the usage of it to calculate the duration in TPQN will be in `MeasureDurationValidator`